checkov/terraform/checks/resource/gcp/GoogleComputeDiskEncryption.py

```diff
@@ -0,0 +1,20 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class GoogleComputeDiskEncryption(BaseResourceCheck):
```

Comment:
@arielkru Can this be refactored into `BaseResourceValueCheck`?

Reply:
the scan is based on `len(conf['disk_encryption_key'][0]) > 0`, which is not an option in `BaseResourceValueCheck`  (if I use ANY_VALUE it would pass with an empty list, which is not the same check)